nurseconnect/templates/base.html

```diff
@@ -69,9 +76,9 @@
         </script>
 
         <!-- this section should always be last in the head -->
-        {% compress css file enhanced %}
+        <!-- {% compress css file enhanced %}
```

Comment:
Remove or use django comment tags.
Do not leave code that will not be re-activated.